examples/feature-factory/package.json

```diff
@@ -19,5 +19,8 @@
     "eslint": "^8.53.0",
     "eslint-config-ponder": "workspace:*",
     "typescript": "^5.2.2"
+  },
+  "engines": {
+    "node": ">=18"
```

Comment:
What's the experience like if this is violated? Do all package managers respect it?